src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const orderedStyles = sourceString.split(';');
+
+  const cssObject = {};
+
+  orderedStyles.forEach((style) => {
+    const [key, value] = style.split(':');
+
+    if (key && value) {
+      cssObject[key.trim()] = value.trim();
+    }
+  });
```

Comment:
[CODE STYLE] - When using chained methods, ensure that they are indented at the same level for better readability. In this case, if you were to chain methods after `split`, each method should start on a new line and aligned with the same indentation.